Gemfile

```diff
@@ -9,6 +9,8 @@ gem "winrm-transport"
 
 group :development do
   gem "kitchen-vagrant"
+  gem "knife-cookbook-doc"
+  gem "travis"
```

Comment:
Add gem to automatically generate the README.md
Also, always install the travis command line tool.